pkiintergration-fe/src/services/create-certificate.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { FetchClient } from '@c8y/client';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CreateCertificateService {
+  private options = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' }
+  };
+
+  constructor(private fetchClient: FetchClient) {}
+  async getCertificate() {
```

Comment:
nitpick: make code more readable

(codestyle) insert a new line between functions to make your code better "scanable"/readable